packaging/nix/flake.nix

```diff
@@ -2,55 +2,57 @@
   description = "umu universal game launcher";
 
   inputs = {
-    nixpkgs = {
-      type = "github";
-      owner = "NixOS";
-      repo = "nixpkgs";
-      ref = "nixpkgs-unstable";
-    };
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
   };
 
   outputs = {
     self,
     nixpkgs,
   }: let
-    umu-launcher-src = builtins.toPath "${self}/../../";
+    inherit (nixpkgs) lib;
 
-    pkgs = import nixpkgs {
-      system = "x86_64-linux";
-      overlays = [self.overlays.default];
-    };
+    # Utility function for producing consistent rename warning messages
+    rename = old: new: lib.warn "`${old}` has been renamed to `${new}`";
 
-    version = "1.1.4";
-  in {
-    overlays.default = final: prev: let
-      py = prev.python3;
-    in {
-      umu-launcher = final.callPackage ./umu-launcher.nix {
-        inherit version;
-        umu-launcher = umu-launcher-src;
-        pyth1 = py;
-      };
+    # Package sets supported by umu-launcher-unwrapped
+    # { system = pkgs; }
+    supportedPkgs =
+      lib.filterAttrs
+      (system: pkgs: builtins.elem system pkgs.umu-launcher-unwrapped.meta.platforms)
+      nixpkgs.legacyPackages;
```

Comment:
I think this is slowing down the flake eval, by causing each nixpkgs instance to be evaluated.

I'll try and come up with a more performant approach, but we may just have to statically define which systems are supported.

Reply:
Resolved using:

```nix
platforms = lib.platforms.linux;
supportedPkgs = lib.filterAttrs (system: _: builtins.elem system platforms) nixpkgs.legacyPackages;
```